android/app/src/androidTest/java/com/microsoft/codepush/common/UpdateManagerTest.java

```diff
@@ -102,11 +95,19 @@ public void setUp() throws Exception {
     @Test
     public void fullWorkflowTest() throws Exception {
         codePushUpdateManager.clearUpdates();
-        executeFullWorkflow(packageObject, codePushUpdateManager, FULL_PACKAGE_HASH, FULL_PACKAGE_URL);
-        executeFullWorkflow(packageObject, codePushUpdateManager, DIFF_PACKAGE_HASH, DIFF_PACKAGE_URL);
-        CodePushLocalPackage codePushPreviousPackage = codePushUpdateManagerDeserializer.getPreviousPackage();
-        CodePushLocalPackage codePushCurrentPackage = codePushUpdateManagerDeserializer.getCurrentPackage();
-        CodePushLocalPackage codePushPackage = codePushUpdateManagerDeserializer.getPackage(DIFF_PACKAGE_HASH);
+        executeFullWorkflow(codePushUpdateManager, FULL_PACKAGE_HASH, FULL_PACKAGE_URL);
```

Comment:
What is actually `FULL_` prefix means?

Reply:
@sergey-akhalkov, have added the explanation.